pkg/cmd/pr/shared/params.go

```diff
@@ -154,6 +154,7 @@ type FilterOptions struct {
 	Labels     []string
 	Author     string
 	BaseBranch string
+	HeadBranch string
```

Comment:
not sure we actually need this changes, it seems it has no any impact on a result, but I did it by analogy with `--base` 🤔

Reply:
Well, it seems it will be used in case of `--web`.